run-in-vm.sh

```diff
@@ -1,30 +1,33 @@
 #!/bin/bash
-set -euo pipefail
+set -x -euo pipefail
 name=${VMNAME:-backend-server}
-test -x "$(command -v multipass)" || { echo "You need multipass installed to run this script: https://multipass.run"; exit 1; }
+test -x "$(command -v multipass)" || { echo "You need multipass installed to run this script: https:/multipass.run"; exit 1; }
```

Comment:
genuine question - is `https:/multipass.run` a typo?

Reply:
it is!